internal/commands/fmt.go

```diff
@@ -44,7 +60,7 @@ func NewFormatCommand() *cobra.Command {
 					return fmt.Errorf("read policy: %w", err)
 				}
 
-				formattedContents, err := format.Source(policy.Package.Location.File, contents)
+				formattedContents, err := format.SourceWithOpts(policy.Package.Location.File, contents, format.Opts{RegoVersion: selectedRegoVersion, ParserOptions: &ast.ParserOptions{RegoVersion: selectedRegoVersion}})
```

Comment:
Is there a scenario in which we would want to parse v0, but fmt with v1? The opa CLI seems to take a more nuanced approach

https://github.com/open-policy-agent/opa/blob/c0871837e886a84867aea652671d94ace15d0068/cmd/fmt.go#L159